wxt_cidrbot/cmd_list.py

```diff
@@ -516,13 +525,19 @@ def help_menu(self, help_type):
             return reminders_help + end_text
         if help_type == "syntax":
             return syntax_help + end_text
+        if help_type == "triage":
+            return triage_help + end_text
         return "No help type found"
 
     def triage_list(self, room_id):
-        triage = self.dynamo.get_triage(room_id)
-        message = "Current list of triage users Cidrbot assigns issues to:\n"
-        for user in triage:
-            message += "- " + user + " \n"
+        try:
```

Comment:
would it make more sense to call these functions something like "list_triage_message" - `triage_list` is fairly ambiguous, and I would personally expect it to be a python list of who can triage

Reply:
I was following the "list_repos" function however list_triage_message better describes what it does